src/main/resources/com/google/api/codegen/csharp/gapic_smoketest_csproj.snip

```diff
@@ -0,0 +1,22 @@
+# packageMetadata: PackageMetadataView
+@snippet generate(packageMetadata)
+  <?xml version="1.0" encoding="utf-8"?>
+  <Project Sdk="Microsoft.NET.Sdk">
+  
+    <PropertyGroup>
+      <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Any reason for targeting 2 rather than 1?

Reply:
No reason. Changed to 1.0